packages/edit-site/src/components/global-styles/variations/style.scss

```diff
@@ -5,34 +5,45 @@
 	cursor: pointer;
 
 	.edit-site-global-styles-variations_item-preview {
-		padding: $border-width * 2;
 		border-radius: $radius-block-ui;
-		box-shadow: 0 0 0 $border-width $gray-200;
 		// Shown in Windows 10 high contrast mode.
 		outline: 1px solid transparent;
+		position: relative;
+		transition: box-shadow 0.1s linear;
+
+		&::after {
+			content: "";
+			position: absolute;
+			top: -1px;
```

Comment:
You can use $border-width instead of 1px. Mainly for legibility.

Reply:
Good look; added via https://github.com/WordPress/gutenberg/pull/60031/commits/1910ef887a1e66e8852a5ca21618acb7bc7865fd.